common/src/main/resources/db/changelog/v2022/metrics_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS event.event_metrics
```

Comment:
This is already in ab2d-events repo (as it should be). Why is it duplicated here?

Reply:
Liquibase refuses to run from ab2d-events so I copied the table here so it at least gets created.
I'll ask @sb-dkniesteadt about it.

Reply:
Event service changelogs only exist for testing. Could be that while it exist in one repo it dosent in another. Not 100% sure

Reply:
It could just be my local setup but if I don't specifically add liquibase core to ab2d-events dependencies in gradle liquibase won't run. Anyways I have a fix now so I'll remove this.